lib/oli/authoring/course/project.ex

```diff
@@ -31,6 +31,8 @@ defmodule Oli.Authoring.Course.Project do
 
     has_many :publications, Oli.Publishing.Publication
 
+    belongs_to :publisher, Oli.Inventories.Publisher, foreign_key: :publisher_id
```

Comment:
NIT: I think the foreign_key is the default

Reply:
💯